README.md

```diff
@@ -31,6 +31,10 @@ npm start
 
 Learn more about Electron and its API in the [documentation](http://electron.atom.io/docs/).
 
+---
+Warning: Windows 10 users might encounter some errors while running `npm start` from Linux Bash for Windows. The reason for this is that Electron is a GUI application, but Linux bash for Windows is just a headless VM without any GUI ( [explained here](https://github.com/electron/electron-quick-start/issues/129#issuecomment-279701745) ).
+A temporary workaround is getting [NodeJS for Windows](https://nodejs.org/en/) and use the node cmd.
```

Comment:
When you install NodeJS for Windows, it also installs a Node.js command prompt. It's like Window's cmd (dir instead of ls) but with node/npm already set up.

It's temporary because Windows 10 is in active development and maybe they change that. I also found out that you can use `export DISPLAY=:0` before `npm start` directly on the Linux bash for Windows and get it to work for a "Hello world" application, but I don't know how it would work on applications with animations and other graphics.